apps/web/src/pages/brand/tabs/LayoutEditor.tsx

```diff
@@ -99,16 +141,16 @@ export function LayoutEditor({ id, goBack }: { id: string; goBack: () => void })
         </Grid>
 
         <Controller
-          name="layout"
+          name="content"
```

Comment:
What about `layout-content` so it is easier to find and then probably for testing?

Reply:
It's the field's name in the form. I want to keep it identical to the fields used in the backend. 
But I will add data-test-id: 'layout-content' for the tests💪🏻